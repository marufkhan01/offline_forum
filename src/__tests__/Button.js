import React from 'react';
import { shallow,mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';

import Button from '../components/Button';


it('simulate click event of button', () => {
    const wrapper = mount(<Button onClick={() => {}} />);
    expect(wrapper.props().danger).toBe(false);
});





it('Button component should render as expected',() =>{
    const component = shallow(<Button />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
});