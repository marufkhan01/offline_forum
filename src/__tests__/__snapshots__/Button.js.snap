// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render as expected 1`] = `
<button
  className="bg-indigo-dark hover:bg-indigo-darker text-white font-bold py-2 px-4 rounded-full "
  data-test="button"
/>
`;
